prime-router/src/main/kotlin/azure/DatabaseLookupTableAccess.kt

```diff
@@ -197,6 +197,8 @@ class DatabaseLookupTableAccess(private val db: DatabaseAccess = DatabaseAccess(
                 }
             } while (dataBatch != null)
         }
+        // refresh the materialized views
+        db.refreshMaterializedViews(tableName)
```

Comment:
Invoke the proc above when a lookup table is updated.

Reply:
What uses the materialized view?  Any chance of interrupting existing processes, or not being able to call this function due to long running queries locking the view?

Reply:
@clediggins-usds nothing is using it right now. I want to add this to Metabase